tests/test_route_b_card_shape_recall.py

```
@@ -0,0 +1,148 @@
+"""Route B 召回：新一代形状的卡必须能被想起来，且正文绝不外泄。
+
+2026-08-16 线上事故（usr_5a8a31b255ecd942）：
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)